examples/webgl_tonemapping.html

```diff
@@ -44,10 +44,14 @@
 				ACESFilmic: THREE.ACESFilmicToneMapping
 			};
 
-			init();
-			render();
+			init()
+				.catch( function ( err ) {
```

Comment:
Minor formatting. It seems you can write this in one line.

Reply:
Yeah, I wasn't sure which way would be preferred. Moved onto one line.